tempo_embeddings/embeddings/vector_database.py

```diff
@@ -0,0 +1,336 @@
+import os, json
+import shutil
+from abc import ABC, abstractmethod
+import numpy as np
+from tqdm import tqdm
+from typing import Iterable, List, Callable, Dict, Any, Optional
```

Comment:
As above, no need to rely on `List` and `Dict` here.
```suggestion
from typing import Iterable, Callable, Any, Optional
```